extensions/css-language-features/client/src/node/cssClientMain.ts

```diff
@@ -37,6 +37,7 @@ export async function activate(context: ExtensionContext) {
 	process.env['VSCODE_L10N_BUNDLE_LOCATION'] = l10n.uri?.toString() ?? '';
 
 	client = await startClient(context, newLanguageClient, { fs: getNodeFSRequestService(), TextDecoder });
+	context.subscriptions.push(registerCssLanguageFeatures());
```

Comment:
Let's check with @aeschli  on where this code should live. Right now it's under `node` so it will only run on desktop VS Code (not browsers)